scripts/src/main/resources/scripts/command/vscode

```diff
@@ -14,27 +14,19 @@ then
   exit
 fi
 
-if [ -n "${DEVON_IDE_TRACE}" ]; then set -vx; fi
-VSCODE_HOME="${DEVON_IDE_HOME}/software/vscode"
-TOOL_VERSION_COMMAND="code -v"
 # shellcheck source=scripts/functions
 source "$(dirname "${0}")"/../functions
+VSCODE_HOME="${DEVON_IDE_HOME}/software/vscode"
+TOOL_VERSION_COMMAND="code -v"
+# shellcheck source=scripts/commandlet-cli
+source "$(dirname "${0}")"/../commandlet-cli
+
 cd "${DEVON_IDE_HOME}" || exit 255
 
 function doSetup() {
-  if [ "${1}" != "silent" ] || [ ! -d "${VSCODE_HOME}" ]
+  doInstall "vscode" "${VSCODE_VERSION}" "${1}"
```

Comment:
It seems that the download url of VS Code for Linux changed, so the installation doesn't work. 
As new source for the download url https://code.visualstudio.com/sha could be an option.

Reply:
For the most recent stable version of VSCode on debian-based linux the url currently looks like this: 
https://code.visualstudio.com/sha/download?build=stable&os=linux-deb-x64